climakitae/util/utils.py

```diff
@@ -448,3 +451,68 @@ def hdh_cdh_lineplot(data):
     return data.hvplot.line(
         x="time", by="simulation", title=data.name, ylabel=data.name + " (degF)"
     )
+
+
+def convert_to_local_time(
+    data: xr.Dataset, selections: Select, lat: float, lon: float
```

Comment:
Can't we get the location information from the data itself? Either with `lat`, `lon` or the station name lookup? Are we trying to convert to the timezone of the data location or to a specific user defined timezone?

Reply:
Intention is to convert from UTC to the local timezone of the selected location. Because parts of our domain are in mountain timezone (and central at 45km), depending on the location, would need to be able to identify the correct timezone

Reply:
Right, so how do we want to deal with data that spans 2 timezones? Just provide a single point like in the code now and use that or base it on cell/station location?

Reply:
This is a good question. For single points/station locations, it's fine to use the timezone that it's in. Both the TMY and heat index applications are for point locations, so that's good. The degree days looks at both a cached area and a point location, and I know the renewables work will need gridded areas, so perhaps whatever the timezone is in the closest-to-central cell is for gridded areas?